ui/app/builds/directives/pnc-log-streamer/pncLogStreamer.js

```diff
@@ -34,7 +34,7 @@
       // --------------------
 
       $ctrl.$onInit = () => {
-        const bifrostUrl = new URL(pncProperties.externalBifrostUrl);
+        const bifrostUrl = new URL(pncProperties.bifrostWsUrl);
```

Comment:
I think `const bifrostWsUrl` should be used too to prevent misunderstanding between ws and http service